forms.py

```diff
@@ -12,14 +12,19 @@
     (7, 'air_pressure'),
     (8, 'snow_amount'),
     (9, 'surface_air_pressure'),
-    (10, None),
+    # (10, None),
     (11, 'surface_runoff_amount'),
     (12, 'precipitation_amount'),
     (13, 'wind_speed'),
     (14, 'surface_roughness_length'),
     (15, 'surface_direct_downwelling_shortwave_flux_in_air')
 ]
 
+TECHNOLOGY_CHOICES = [
```

Comment:
@christian-rli  : here I am not sure what the full options of the feedinlib are

Reply:
For now we should only add what we can expect to see soon in the database, and that is an untested sample dataset covering wind for the year 2016 only. I'm going to upload it today. That also means, for the time being, we don't have to add any forms, but we can just display wind feed-in for that year in every region.